Inicio de sesion implementado

nico-alvz opened the pull request.

nico-alvz committed:
Merge pull request #3 from nico-alv/test

Test

luciano90a committed:
session

CL001-002 V1

luciano90a committed:
CL001-002

SESSION-REGISTER

NecroHunterr commented on app/Http/Controllers/Sessioncontroller.php:
Nombre del archivo no sigue el estandar

luciano90a committed:
refactor: eliminado AdminAuth.php  y cambios en login

- se elimino el archivo adminauth ya que se implemento con rutas y era codigo basura
- se agrego cuadro a login
- se elimino referencia en kernel.php la linea 57

nico-alv commented:
Cerrado por nuevos cambios del autor
